### PR TITLE
Adds dark mode button

### PR DESCRIPTION
**Pull request by Arafat008**
I have added Dark Mode Button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,10 @@
 	</head>
 
 	<body>
+		<div class="darkmode">
```
Comment (Nikhossain) [inline index.html:9]: Must have a button to enable light mode

**Review by Nikhossain — CHANGES_REQUESTED** [review]
Please Implement the light mode button

File: index.html (hunk 1 above)
Comment (Arafat008) [reply index.html:9]: @Nikhossain I have added  light mode button

**Review by Arafat008 — COMMENTED** [review]
I have added light mode button

**Review by Nikhossain — APPROVED** [review]

**Pull request merged**